### DISCUSSION TITLE
[Aesthetic] HD Shield Banner Patterns

### DISCUSSION THREAD

**Discussion by NSPC911** [BedrockTweaks/Files#90] [Ideas]
Interestingly was not in [issues](https://github.com/BedrockTweaks/Files/issues) even though it is definitely a possible pack but there is no trace of it in [packs.json](https://github.com/BedrockTweaks/Files/blob/devel/resource_packs/packs.json).

**Comment by drav0011** [comment]
Totally forgot about it, moving this to issues
